src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+body {
+  font-family: ' Avenir ', sans-serif;
```

Comment:
There are extra spaces inside the quotes of the font-family name 'Avenir'. It should be written without leading or trailing spaces as `font-family: 'Avenir', sans-serif;`.